airflow/ti_deps/deps/ready_to_reschedule.py

```diff
@@ -40,7 +40,9 @@ def _get_dep_statuses(self, ti, session, dep_context):
         considered as passed. This dependency fails if the latest reschedule
         request's reschedule date is still in future.
         """
-        if not getattr(ti.task, "reschedule", False):
+        is_mapped = ti.task.is_mapped
+        if not is_mapped and not getattr(ti.task, "reschedule", False):
+            # Mapped sensors don't currently have the reschedule property,
```

Comment:
```suggestion
            # Mapped sensors don't have the reschedule property (it can only
            # be calculated after unmapping), so we don't check them here.
            # They are handled below by checking TaskReschedule instead.
```